resources/runners/README.md

```diff
@@ -1,6 +1,29 @@
+# NodeOS
 This zip file host a pre-build NodeOS instance with a demo users filesystem, you
 can find info about how to build your own users filesystem at
 [NodeOS-usersfs](node_modules/nodeos-usersfs/README.md) package.
 
-To run this demo, simply exec the ```run.sh``` script. You'll need to have
-[QEmu](http://wiki.qemu.org/Main_Page) installed.
+## Getting started
+This pre-build requires [QEmu](http://wiki.qemu.org/Main_Page).
+Please make sure that it's installed.
+
+To run this demo, simply exec the ```run.sh``` script in the root directory.
+
+You should be greeted by the NodeOS-usersfs.
+It will ask you for a username and a password.
+```
+$ username: nodeos
+$ password: nodeos
+```
+If you run into trouble, please checkout the [troubleshooting here](https://github.com/NodeOS/NodeOS/wiki/Troubleshooting)
+
+## Burning to CD or USB
```

Comment:
Since the ISO has specific instructions that don't apply to others (QEmu, for example), then it's better to copy this file to `README_iso.md`, put that burning instructions there, and update the `postinstall` script to use that file instead for ISO images. The same would need to be done for disk images.